cypress/integration/getTokenSilently.js

```diff
@@ -55,7 +55,7 @@ describe('getTokenSilently', function() {
           cy.toggleSwitch('local-storage');
 
           cy.login().then(() => {
-            cy.reload();
+            cy.reload().wait(5000);
```

Comment:
Why is there a new 5 second delay here?

Reply:
This test intermittently fails because the UI isn’t ready yet, I’m hoping the delay will fix that. Seems to in the tests I’ve run locally, will see how it pans out over time.